api/src/wfl/source.clj

```diff
@@ -157,8 +157,8 @@
   "Query TDR for rows within `_interval` that are new to `source`."
   [{:keys [dataset details table column] :as source}
    [begin end                            :as _interval]]
-  (log/debug (format "%s Looking for rows in %s.%s between [%s, %s]..."
-                     (log-prefix source) (:name dataset) table begin end))
+  (log/info (format "%s Looking for rows in %s.%s between [%s, %s]..."
```

Comment:
This is now a more remarkable condition since it's happening only every 20 minutes, so bumped it up to INFO.